tlatools/org.lamport.tlatools/src/util/Set.java

```diff
@@ -375,18 +376,21 @@ public void intersection (Set x) {
   }
 
   /**
-   * Create a new set which is the intersection of x and y.  We clone
-   * the smaller (set size) of x, y because that's what intersection
-   * iterates on.
+   * Create a new set which is the intersection of x and y.  We make
+   * a shallow copy of the smaller (set size) of x, y because that's
+   * what intersection iterates on.
+   *
+   * TODO 2023/10/23: the comment above is wrong; the branches are identical.
+   *     We are missing out on whatever performance benefits could be gained.
    */
   public static Set intersection(Set x, Set y) {
```

Comment:
All the static variants of intersection, minus, complement, ... appear to be dead code anyway.